### ISSUE TITLE
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

### ISSUE THREAD

**Issue by ruslanguns** [angular/angular-cli#15242]
```bash
$ ng serve

i ｢wds｣: Project is running at http://localhost:4200/webpack-dev-server/
i ｢wds｣: webpack output is served from /
i ｢wds｣: 404s will fallback to //index.html
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

<--- Last few GCs --->

[15508:0000017B4CD5C0A0]    68631 ms: Mark-sweep 1986.4 (2066.6) -> 1972.4 (2068.1) MB, 1755.4 / 0.0 ms  (average mu = 0.137, current mu = 0.060) allocation failure scavenge might not succeed
[15508:0000017B4CD5C0A0]    70523 ms: Mark-sweep 1987.7 (2068.1) -> 1973.5 (2069.6) MB, 1789.9 / 0.0 ms  (average mu = 0.095, current mu = 0.054) allocation failure scavenge might not succeed


<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 00007FF691272C16]
Security context: 0x006e7dc1a2f1 <JSObject>
    1: _makeAsyncAwaitCompleter [0000028A8AF5EA89] [D:\wamp\www\proyectos\1-proyectos_integralclick\ic_core-client\node_modules\sass\sass.dart.js:~3190] [pc=0000019C2623FFA2](this=0x02f35c0309c1 <Object map = 000001AC1AFBFB59>,0x028a8af63ec9 <JSFunction String (sfi = 00000279470DBCF1)>)
    2: /* anonymous */ [000001C9474F3299] [D:\wamp\www\proyectos\1-pro...


Writing Node.js report to file: report.20190802.224642.15508.0.001.json
Node.js report completed
 1: 00007FF69072674F napi_wrap+120927
 2: 00007FF6906CFE26 uv_loop_fork+44550
 3: 00007FF6906D0CC6 uv_loop_fork+48294
 4: 00007FF690A08BCE private: void __cdecl v8::internal::Parser::ReportMessageAt(struct v8::internal::Scanner::Location,enum v8::internal::MessageTemplate,class v8::internal::AstRawString const * __ptr64,enum v8::internal::ParseErrorType) __ptr64+206
 5: 00007FF6909DCC4A public: void __cdecl v8::internal::wasm::StreamingDecoder::NotifyCompilationEnded(void) __ptr64+666
 6: 00007FF690A7309C public: class v8::internal::HeapObject __cdecl v8::internal::Heap::CreateFillerObjectAt(unsigned __int64,int,enum v8::internal::ClearRecordedSlots,enum v8::internal::ClearFreedMemoryMode) __ptr64+2972
 7: 00007FF690A7AE0F public: void __cdecl v8::internal::LocalEmbedderHeapTracer::NotifyV8MarkingWorklistWasEmpty(void) __ptr64+5375
 8: 00007FF690A71483 public: bool __cdecl v8::internal::Heap::CollectGarbage(enum v8::internal::AllocationSpace,enum v8::internal::GarbageCollectionReason,enum v8::GCCallbackFlags) __ptr64+1235
 9: 00007FF690A6FB84 public: void __cdecl v8::internal::GCTracer::AddScopeSample(enum v8::internal::GCTracer::Scope::ScopeId,double) __ptr64+1316
10: 00007FF690AB07B3 public: class v8::internal::Handle<class v8::internal::HeapObject> __cdecl v8::internal::Factory::NewFillerObject(int,bool,enum v8::internal::AllocationSpace) __ptr64+67
11: 00007FF690DD76E6 public: void __cdecl v8::internal::OptimizingCompileDispatcher::Unblock(void) __ptr64+116966
12: 00007FF691272C16 public: static class v8::internal::Vector<char const > __cdecl v8::internal::NativesCollection<0>::GetScriptsSource(void)+662454
```
There's the output... 

# 🐞 Bug report

### Command (mark with an `x`)

```
- [ ] new
- [ ] build
- [x] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Description
 A clear and concise description of the problem...

Hello folks. 
Today I tried to rebuild my repository and make a ng serve as commonly but I just figured out that there's an issue with node_modules/sass/sass.dart.js on compiling.

## 🌍 Your Environment
<pre><code>
* Angular CLI: 8.2.0
* Node: 12.3.1
* OS: win32 x64
* Angular: 8.2.0
</code></pre>

**Comment by ChristopherKiss** [comment]
This looks like a duplicate of the stickied issue https://github.com/angular/angular-cli/issues/13734

**Comment by alan-agius4** [comment]
Thanks @ChristopherKiss, indeed.
